script.matchcenter/addon.xml

```diff
@@ -12,7 +12,7 @@
 		<platform>all</platform>
 		<website>http://www.thesportsdb.com/</website>
 		<source>https://github.com/enen92/script.matchcenter</source>
-		<forum>http://forum.kodi.tv/showthread.php?tid=238039</forum>
+		<forum>http://forum.kodi.tv/showthread.php?tid=259894</forum>
 		<summary lang="en">Extended football match information</summary>
 		<summary lang="pt">Informação extra para encontros de futebol</summary>
```

Comment:
I've just read about it on the wiki. Should it contain the information about every version por just the for the last update?